src/nft-card.ts

```diff
@@ -20,8 +25,24 @@ import './nft-card-back.ts'
 @customElement('nft-card')
 export class NftCard extends LitElement {
 
-  static get styles() {
-    return css`
+    /* User configurable properties */
+    @property({type: Boolean}) public horizontal = false
+    @property({type: String}) public contractAddress
+    @property({type: String}) public tokenId
+    @property({type: String}) public width = this.horizontal ? '670px' : '388px'
+    @property({type: String}) public height = this.horizontal ? '250px' : '560px'
+    @property({type: String}) public network: Network = Network.Main
+    @property({type: Object}) private asset
+    @property({type: Object}) private traitData = {}
+    @property({type: String}) private account
+    @property({type: String}) private flippedCard = false
+    @property({type: Boolean}) private loading = true
+    @property({type: Object}) private provider = ('web3' in window) ? window.web3.currentProvider : new Web3.providers.HttpProvider('https://mainnet.infura.io')
```

Comment:
i just realized that i'm not sure web3.currentProvider can be relied on - I thought i read that `window.ethereum` was the new one, and the former just exists for legacy reasons